.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - "8"
+  - "9"
+install:
```

Comment:
can you also add test stage here running `npm run test`

Reply:
Added after npm run build